app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/models/EventShowcaseModel.java

```diff
@@ -42,6 +41,17 @@ public void init(int eventId) {
         return event;
     }
 
+    public LiveData<EventLocation> getLocation() {
```

Comment:
I don't think this method has a valid reason to exist, we can directly call ev.getLocation() from a subscriber

Reply:
Solved in 691c9df